models/nft/schema.yml

```diff
@@ -9,3 +9,5 @@ models:
       materialized: ephemeral
   - name: nft_trades
   - name: nft_trades_partition_by_dt
+  - name: whale
+  - name: smart_money
```

Comment:
I recommend to put all the labels into a dedicated folder alongside with all the other labels (like dex trader) in the future. e.g.

labels/smart_nft_trader
labels/nft_whale
labels/smart_dex_trader